django/bossspatialdb/views.py

```diff
@@ -388,7 +388,7 @@ def post(self, request, collection, experiment, channel):
             'res_lt_max': int(channel.base_resolution) + 1 < int(experiment.num_hierarchy_levels),
 
             # DP NOTE: hardcode for the moment, users will expect not all resolutions will be indexed
-            'annotation_index_max': 3, # Create index for resolutions  1, 2 (Resolution 0 should already exist)
+            'annotation_index_max': 1, # Create index for resolutions  1, 2 (Resolution 0 should already exist)
```

Comment:
Three issues
1) this will disable annotation indexing within the downsampling process
2) the attached comment should have been updated, as it is now confusing
3) the change doesn't match the commit message, make another source of confusion